Examples/Tests/silver_mueller/analysis_silver_mueller_rz.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+# Copyright 2021
+#
+# This file is part of WarpX.
+#
+# License: BSD-3-Clause-LBNL
+"""
+This tests verifies the Silver-Mueller boundary condition.
+A laser pulse is emitted and propagates towards the boundaries ; the
+test check that the reflected field at the boundary is negligible.
+"""
+
+import os
+import sys
+
+import numpy as np
+
+import yt ; yt.funcs.mylog.setLevel(0)
+sys.path.insert(1, '../../../../warpx/Regression/Checksum/')
+import checksumAPI
+
+filename = sys.argv[1]
+
+ds = yt.load( filename )
+all_data_level_0 = ds.covering_grid(level=0,left_edge=ds.domain_left_edge, dims=ds.domain_dimensions)
+Er = all_data_level_0['boxlib', 'Er'].v.squeeze()
+Et = all_data_level_0['boxlib', 'Et'].v.squeeze()
+Ez = all_data_level_0['boxlib', 'Ez'].v.squeeze()
+# The peak of the initial laser pulse is on the order of 6 V/m
+# Check that the amplitude after reflection is less than 0.01 V/m
+max_reflection_amplitude = 0.01
+assert np.all( abs(Er) < max_reflection_amplitude )
+assert np.all( abs(Et) < max_reflection_amplitude )
+assert np.all( abs(Ez) < max_reflection_amplitude )
```

Comment:
Is this analysis here significantly different from what was done in `analysis_silver_mueller.py` (the original Cartesian version)? If not, we could simply have an `if` condition in the original analysis script, to distinguish the Cartesian and RZ cases. And we could parse the output file `warpx_used_inputs` to learn which test is being run (this is already done in other analysis scripts). If the differences are minimal, I would be in favor of not duplicating the analysis script. Especially because we don't do it consistently throughout the WarpX test suite (we don't enforce the policy "one test - one input - one analysis script", it was discussed in the past, but never done).

Reply:
The analysis is not significantly different, this is a good idea.  I did not think to use ``warpx_used_inputs`` to do logic on the geometry. Thanks, @EZoni !